rocketpy/rocket/aero_surface/fins/fins.py

```diff
@@ -437,11 +437,17 @@ def to_dict(self, include_outputs=False):
             "name": self.name,
         }
 
-        if include_outputs:
+        if kwargs.get("include_outputs", False):
+            cl = self.cl
+            if kwargs.get("discretize", False):
+                cl = cl.set_discrete(
```

Comment:
@MateusStano , could you give some insights if these boundary values make sense (angle of attack from -15 to 15 and mach 0 to 2)?